grpc-protocol/src/main/java/com/linecorp/armeria/server/grpc/protocol/AbstractUnsafeUnaryGrpcService.java

```diff
@@ -85,17 +124,24 @@ protected final HttpResponse doPost(ServiceRequestContext ctx, HttpRequest req)
                         return handleMessage(ctx, requestMessage);
                     }
                 }).thenApply(responseMessage -> {
+                    final HttpHeadersBuilder trailersBuilder = HttpHeaders.builder();
+                    GrpcTrailersUtil.addStatusMessageToTrailers(trailersBuilder, StatusCodes.OK, null);
+                    final HttpHeaders trailers = trailersBuilder.build();
```

Comment:
This is not related to this change, but how about adding trailers to the `RequestContext` such as we do in `ArmeriaServerCall` https://github.com/line/armeria/blob/master/grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java#L383
so that `MeterIdPrefixFunction` can use the status? https://github.com/line/armeria/blob/master/grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcMeterIdPrefixFunction.java#L108

Reply:
Thanks for the suggestion, @minwoox! To do that I'll need to move GrpcWebTrailers from com.linecorp.armeria.common.grpc to com.linecorp.armeria.common.grpc.protocol (because com.linecorp.armeria.common.grpc.protocol cannot depend on com.linecorp.armeria.common.grpc). Not sure if that's OK because it's a public class in a public part of the package. 

Also, if we move GrpcWebTrailers to com.linecorp.armeria.common.grpc.protocol - will it be confusing that GrpcMeterIdPrefixFunction is not available in com.linecorp.armeria.common.grpc.protocol? It seem to depend on io.grpc, so we cannot move it to com.linecorp.armeria.common.grpc.protocol too.

If we're not sure - I'd suggest to file a feature request and follow up on this later. I'd prefer to keep this PR focused on one thing.

Reply:
> To do that I'll need to move GrpcWebTrailers from com.linecorp.armeria.common.grpc to com.linecorp.armeria.common.grpc.protocol

I think it's fine because `GrpcWebTrailers` is an unstable API.
Or we could only move the `AttributeKey` inside of `GrpcWebTrailers` https://github.com/line/armeria/blob/master/grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcWebTrailers.java#L38 to `grpc-protocol` and use it so that we don't have to move `GrpcWebTrailers`.

> will it be confusing that GrpcMeterIdPrefixFunction is not available in com.linecorp.armeria.common.grpc.protocol?

Same here. We don't have to move it.

> I'd prefer to keep this PR focused on one thing.

It's fine for me as well. 😄 Let's solve the issue in a separate PR if it's preferred. 😄

Reply:
Sounds good, created https://github.com/line/armeria/issues/3724.